xwiki-platform-core/pom.xml

```diff
@@ -125,9 +125,18 @@
 
                  Single justification example:
             -->
-            
-            
-            
+            <revapi.differences>
+              <justification>Needed changes for properly handling merge with fallback version.</justification>
+              <criticality>highlight</criticality>
+              <differences>
+                <item>
+                  <ignore>true</ignore>
+                  <code>java.method.addedToInterface</code>
+                  <new>method org.xwiki.store.merge.MergeManagerResult&lt;com.xpn.xwiki.objects.ElementInterface, java.lang.Object&gt; com.xpn.xwiki.objects.ElementInterface::merge(com.xpn.xwiki.objects.ElementInterface, com.xpn.xwiki.objects.ElementInterface, com.xpn.xwiki.doc.merge.MergeConfiguration, com.xpn.xwiki.XWikiContext)</new>
+                  <justification>Change needed to provide a merge operation taking into account the configuration: we cannot provide a default method as it would wrongly give the feeling that the operation succeeded.</justification>
```

Comment:
I believe we need to explain why we're ok to break it. And the main reason is because this is an API that's unlikely to be extended.

Reply:
Yes that's also @tmortagne feedback in the thread above. I'll add a note.